msteams-platform/index.yml

```
@@ -14,100 +15,120 @@ metadata:
    items: 
-    - title: New to Microsoft Teams development? 🚀Start here! 
+    - title: New to Teams development? 🚀Start here! 
       itemType: get-started
-      url:   /microsoftteams/platform/overview
-    - title:  Tabs — embedded web experiences
-      itemType:  overview
-      url:   /microsoftteams/platform/tabs/what-are-tabs
-    - title:  Bots — conversation-based interactions
-      itemType:  overview
-      url:  /microsoftteams/platform/bots/what-are-bots
-    - title:  Messaging extensions — task oriented interactions
-      itemType: overview
-      url:  /microsoftteams/platform/messaging-extensions/what-are-messaging-extensions
-    - title: Task modules and cards — rich messaging and notifications
-      itemType: overview
-      url:  /microsoftteams/platform/task-modules-and-cards/what-are-task-modules  
-    - title: Webhooks and connectors — programmed messaging and notifications
+      url: /microsoftteams/platform/overview
+    - title: Plan your app
       itemType: overview
-      url:  /microsoftteams/platform/webhooks-and-connectors/what-are-webhooks-and-connectors 
-    - title:  Microsoft Teams and Microsoft Graph API
+      url: /microsoftteams/platform/concepts/design/understand-use-cases
+    - title: Design your app
       itemType: overview
+      url: /microsoftteams/platform/concepts/design/design-teams-app-overview
+    - title: Build your app 
+      itemType: how-to-guide
+      url: /microsoftteams/platform/overview
+    - title: Enhance your app with Microsoft Graph
+      itemType: how-to-guide
       url:  /graph/teams-concept-overview
-    - title:  Low-code apps, bots, flows, and reports using Microsoft Power Platform
-      itemType: overview
-      url:  /microsoftteams/platform/samples/teams-low-code-solutions/
+    - title: Deploy and publish your app
+      itemType: deploy
+      url: /microsoftteams/platform/concepts/deploy-and-publish/appsource/publish
+    - title: Tutorials and code samples
+      itemType: sample
+      url:  /microsoftteams/platform/tutorials/code-samples
+    - title: What's new?
+      itemType: whats-new
+      url: /microsoftteams/platform/whats-new
+
    
 # conceptualContent section
 conceptualContent:
 # itemType: architecture | concept | deploy | download | get-started | how-to-guide | learn | overview | quickstart | reference | sample | tutorial | video | whats-new
    title: Guides and Resources
    items:
-   # Card
-    - title: Explore
+
+  # Card
+    - title: Audience
       links:
-        - url: /microsoftteams/platform/concepts/building-an-app
+        - url: /microsoftteams/microsoft-teams
           itemType: concept
-          text: Build a Microsoft Teams app
-        - url: /microsoftteams/platform/concepts/extensibility-points
+          text: Developers
+        - url: /microsoftteams/?view=msteams-client-js-latest
```

```suggestion
        - url: https://developer.microsoft.com/microsoft-teams
```
Maybe instead link to the marketing dev center page? The JS SDK is kind of limited in that it is mostly for tab development.